### ISSUE TITLE
missing native/lib/win64 folder?

### ISSUE THREAD

**Issue by increpare** [larsiusprime/SteamWrap#31]
setup.bat script assumes it exists, but when you clone the repo it's not there.

**Comment by r2d2rigo** [comment]
@increpare you are right, we hit the same issue while trying to do a recent build for a game. I'm fixing it in a PR.
